Formula/haskell-language-server.rb

```diff
@@ -21,26 +22,53 @@ class HaskellLanguageServer < Formula
 
   depends_on "cabal-install" => [:build, :test]
   depends_on "ghc" => [:build, :test]
+  depends_on "ghc@8.6" => [:build, :test]
+  depends_on "ghc@8.8" => [:build, :test]
 
   def install
     system "cabal", "v2-update"
-    system "cabal", "v2-install", *std_cabal_v2_args
+
+    %w[ghc@8.6 ghc@8.8 ghc].each do |ghc_str|
+      ghc = Formula[ghc_str]
+      ghc_version = ghc.version
```

Comment:
I wonder if this should just be `version.major_minor`, since we don't really need the patch version to be in the language server name. Also I didn't notice you just actually needed the version, so you don't need to assign to this variable, actually. (Wasn't paying attention as usual) You could just do
```ruby
bin.install bin/"haskell-language-server" => "haskell-language-server-#{ghc.version}"
```
below. (or `#{ghc.version.major_minor}`)

Reply:
I would leave the `version`, since this is how HLS does their release, and users can tell at first glance what GHC version this binary is compiled for (since it matters all the way until patch version). The wrapper would manage either way, since it tries to find the matching binary from patch version, through minor and major until no version in the file name.